src/fuel-vm/instruction-set.md

```diff
@@ -710,19 +710,19 @@ Bits     | Short name | Description
 ---------|------------|-------------------------------------
 `...XXX` | `mode`     | Compare mode selection
 `.XX...` | `reserved` | Reserved and must be zero
-`X.....` | `indirect` | Is rhs operand ($rC) indirect or not
+`X.....` | `indirect` | Is rhs operand (`$rC`) indirect or not
 
 Then the actual operation that's performed:
 
 `mode` | Name | Description
 -------|------|-------------------------------------------------------------
-0      | eq   | Equality (`==`)
-1      | ne   | Inequality (`!=`)
-2      | lt   | Less than (`<`)
-3      | gt   | Greater than (`>`)
-4      | lte  | Less than or equals (`<=`)
-5      | gte  | Greater than or equals (`>=`)
-6      | lzc  | Leading zero count the lhs argument (`lzcnt`). Discards rhs.
+0      | `eq`   | Equality (`==`)
+1      | `ne`   | Inequality (`!=`)
+2      | `lt`   | Less than (`<`)
+3      | `gt`   | Greater than (`>`)
+4      | `lte`  | Less than or equals (`<=`)
+5      | `gte`  | Greater than or equals (`>=`)
+6      | `lzc`  | Leading zero count the lhs argument (`lzcnt`). Discards rhs.
 7      | -    | Reserved and must not be used
```

Comment:
nit: Breaks spacing with the rest of the table. No sure if that's worth keeping.

Reply:
Do you mean it doesn't align with the header?

Reply:
Exactly